src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,167 @@
+
+![Provider Details Screenshot](images/authentication/create-oidc-provider-firststep-screenshot.png)
+
+On the "Provider" step, you need to specify the **Issuer URL** of your OIDC provider. This URL points to the OIDC Discovery Document of that particular provider, and GameFabric uses this Discovery Document for integrating with that provider.
+
+**Issuer URL** must only point to the exact discovery endpoint, similar to the examples below; and must NOT include any trailing path (such as "/.well-known/openid-configuration" etc.)
+
+Some **Issuer URL**s are:
+
+| Provider                 | IssuerURL                                               |
+|--------------------------|---------------------------------------------------------|
+| Microsoft ENTRA          | https://login.microsoftonline.com/{tenantID}/v2.0       |
+| Google Identity Platform | https://accounts.google.com                             |
+| Okta                     | https://{yourOktaDomain}/oauth2/default                 |
+| Auth0                    | https://{yourTenant}.auth0.com/                         |
+| Amazon Cognito           | https://cognito-idp.{region}.amazonaws.com/{userPoolId} |
+
+GameFabric must be configured as an external application in the OIDC Provider in order for the provider to assign the [Client ID & Secret](https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/) for GameFabric.
+
+Fill in the **Client ID & Secret** to their respective text inputs, and click "Next".
+
+![Issuer Configuration Screenshot](images/authentication/create-oidc-provider-secondstep-screenshot.png)
+
+For OIDC Providers with standard behavior no further configuration is necessary and you may click "Create" button to finish adding your provider.
+
+![Basic Provider Create Screenshot](images/authentication/create-oidc-provider-basic-finish.png)
+
+### Advanced Configurations
+
+This section allows configuration of **Scopes & Claims**, **Prompt**, **Claim Mapping** and **Provider Discovery Override**.
+
+#### Scopes & Claims
+
+GameFabric requests additional data about the authenticated user using **Scopes & Claims**.
+
+For example, you may want to use a different email field as the users display email on GameFabric, or a different username than the one your provider forwards you by default.
+
+This is accomplished via requesting **Scopes** ([OpenID Scopes](https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes)) from the OIDC Provider.
+
+As a result, OIDC Provider will return **Claims** that contains information about the authenticated user.
+
+**Scopes** text input can be used for requesting these claims during the authentication process. GameFabric will request these **Scopes** from the OIDC Provider.
+
+![Scopes Screenshot](images/authentication/create-oidc-provider-thirdstep-screenshot.png)
+
+ Common scopes include:
+
+| Scope         | Description                                                               |
```

Comment:
Sorry for the last minute request. Could you please add the `offline_access` scope to the table to empathize that it's very important for things to work properly. (Refresh tokens / refreshing the token)